control-plane/grpc/src/operations/volume/traits_snapshots.rs

```diff
@@ -276,13 +276,12 @@ impl VolumeSnapshotState {
     pub fn timestamp(&self) -> Option<&prost_types::Timestamp> {
         self.timestamp.as_ref()
     }
-    /// Get the volume snapshot readiness to clone.
-    pub fn clone_ready(&self) -> bool {
-        match self.repl_snapshots.as_slice() {
-            [VolumeReplicaSnapshotState::Online { state, .. }] => state.clone_ready(),
-            // todo: handle more than one replica for multi-replica snapshots
+    /// Get the volume snapshot readiness to be used as a source.
+    pub fn ready_as_source(&self) -> bool {
+        self.repl_snapshots.iter().any(|snapshot| match snapshot {
```

Comment:
please keep this as it was, we don't support more than 1 replica snapshot clone even when clone drops

Reply:
Okay.